src/etc/inc/upgrade_config.inc

```diff
@@ -737,7 +737,7 @@ function upgrade_040_to_041() {
 		$config['sysctl']['item'][15]['value'] =   "default";
 
 		$config['sysctl']['item'][16]['tunable'] = "net.inet.tcp.inflight.enable";
-		$config['sysctl']['item'][16]['descr'] =    gettext("The system will attempt to calculate the bandwidth delay product for each connection and limit the amount of data queued to the network to just the amount required to maintain optimum throughput. ");
+		$config['sysctl']['item'][16]['descr'] =    gettext("The system will attempt to calculate the bandwidth delay product for each connection and limit the amount of data queued to the network to just the amount required to maintain optimum throughput.");
```

Comment:
I don't like any of these in this file: very very old compatibility layer with m0n0wall creating unnecessary work for translators. If anything, we should strip all gettext() here.